### DISCUSSION TITLE
Error: Please update your system Git

### DISCUSSION THREAD

**Discussion by sunrisewestern** [Homebrew/discussions#2215] [Linux]
I am using linuxbrew on the server with no sudo permission. The git and curl version is too old so I installed them in my home path and managed to install linuxbrew. But when I use brew, the errors kept pop up.

> Error:` Please update your system curl.
Minimum required version: 7.41.0
Your curl version: 7.29.0
Your curl executable: /usr/bin/curl
Error: Please update your system Git.
Minimum required version: 2.7.0
Your Git version: 1.8.3.1
Your Git executable: /usr/bin/git
Unknown option: -C

Is there any way to make brew use git and curl installed in a custom path?

**Comment by carlocab** [comment]
This is experimental, but I believe you should be able to set:
```
HOMEBREW_DEVELOPER=1
HOMEBREW_GIT_PATH=/path/to/your/git
HOMEBREW_CURL_PATH=/path/to/your/curl
```
